test/basic/init_funcs.c

```diff
@@ -0,0 +1,16 @@
+#include "smack.h"
+
+// @expect verified
+
+void __SMACK_init_func_test() {
```

Comment:
Hey, here is an idea, and this might look nicer.
Could you introduce a macro called __SMACK_INIT in smack.h that prefixes whatever it is applied to with __SMACK_init_func_?
So then this function declaration would look like this:
void __SMACK_INIT(test)() {
....
}
You can play with this a little bit...
Another option would be to just have
__SMACK_INIT(test) {
...
}
and then expand it into
void __SMACK_init_func() {
...
}